dropbox.in

```diff
@@ -1083,20 +1038,20 @@ Prints out a public url for FILE (which must be in your public folder).
     try:
         with closing(DropboxCommand()) as dc:
             try:
-                console_print(dc.get_public_link(path=unicode_abspath(args[0].decode(sys.getfilesystemencoding()))).get(u'link', [u'No Link'])[0])
-            except DropboxCommand.CommandError, e:
-                console_print(u"Couldn't get public url: " + str(e))
-            except DropboxCommand.BadConnectionError, e:
-                console_print(u"Dropbox isn't responding!")
+                console_print(dc.get_public_link(path=os.path.abspath(args[0])).get(u'link', [u'No Link'])[0])
```

Comment:
Can `'link'` and `'No Link'` be used without `u` modifier now? I guess Python 3 makes it easier for all strings...

Reply:
indeed, the `u` modifier was removed all over the commit.

Reply:
Thanks @monsta I'll address this in a new PR ASAP!

Reply:
Fixed by #69